js/ui/payg.js

```diff
@@ -456,6 +457,52 @@ function timeToString(seconds) {
     return ("%s %s").format(daysStr, hoursStr);
 }
 
+// Similar to timeToStrings, but does not process partial time,
+// since it's meant to be used for expiration time changes, with
+// a wider range of periods
+function successMessage(seconds) {
+
+    let messageStr = null;
+
+    if (seconds < 60) {
+        messageStr = Gettext.ngettext("%s second has", "%s seconds have", seconds).format(Math.floor(seconds));
+        return _("%s been added to your %s credit.").format(messageStr, NOTIFICATION_TITLE_TEXT);
```

Comment:
You can't break the translatable string in two like this. It needs to be something like:

```js
return Gettext.ngettext(
    "%s second has been added to your Pay As You Go credit.",
    "%s seconds have been added to your Pay As You Go credit.",
    Math.floor(seconds));
```

And the same for all the others. There may be languages where you can't just translate that bit in isolation.